.ci/jobDSL/gradle.properties

```diff
@@ -1,2 +1,2 @@
-jobDslVersion=1.76
-jenkinsVersion=2.190.2
+jobDslVersion=1.77
+jenkinsVersion=2.265
```

Comment:
we cannot bump the version far from 2.255 see https://groups.google.com/g/jenkinsci-dev/c/49gQ-7SYNOc